packages/store/internals/src/internal-tokens.ts

```diff
@@ -0,0 +1,15 @@
+import { InjectionToken } from '@angular/core';
+
+/**
+ * @see only use to access internal {@link StateContextFactory} in plugins
+ */
+export const NGXS_INTERNAL_CONTEXT_FACTORY_TOKEN: InjectionToken<string> = new InjectionToken(
```

Comment:
Not sure about conventions. But see as Angular team does it https://github.com/angular/angular/blob/f8096d499324cf0961f092944bbaedd05364eea1/packages/platform-webworker/src/web_workers/shared/api.ts#L11. They have `TOKEN_NAME = new InjectionToken('tokenName')`. So we could also have `NGXS_STATE_CONTEXT_FACTORY = new Token('Internals.StateContextFactory')`. 

This will ensure consistency. Also we do not come up with our own approach and do as framework creators.

Reply:
@markwhitfeld what do you think? I did according to the example

![image](https://user-images.githubusercontent.com/12021443/65946688-5ca68400-e43f-11e9-8cc3-a835338753b7.png)

Reply:
We cannot change those tokens descriptions, but we can start doing correctly in the new package (by “new” I mean that the internals package doesn’t have any tokens yet).

Reply:
It looks like none of the tokens with the `_TOKEN` suffix are currently exported so we are safe to change our conventions (the others can be fixed in another PR).
@splincode could you change these new token to follow the Angular convention? Sorry to mess you around like this

Reply:
done